src/read.c

```diff
@@ -4403,23 +4413,12 @@ static avifResult avifMetaFindAlphaItem(avifMeta * meta,
     assert(alphaItemCount == colorItemCount);
     // Find an unused ID.
     avifResult result;
-    if (meta->items.count >= UINT32_MAX - 1) {
-        // In the improbable case where all IDs are used.
+    const uint32_t lastID = meta->items.count ? meta->items.item[meta->items.count - 1]->id : 0;
```

Comment:
Nit/optional: Update the "Find an unused ID" comment? Now we aren't really finding an unused ID.

Reply:
Done